package.json

```diff
@@ -1,45 +1,25 @@
 {
   "name": "page-metadata-parser",
+  "description": "A JavaScript library for parsing metadata in a Web Page.",
   "version": "0.1.0",
-  "description": "A Javascript library for parsing metadata in a Web Page.",
-  "main": "parser.js",
-  "scripts": {
-    "test:lint": "eslint ./*.js ./tests/*.js",
-    "test:karma": "karma start",
-    "test:mocha": "./node_modules/istanbul/lib/cli.js cover _mocha --report lcovonly -- tests/*.test.js -R spec",
-    "test": "npm-run-all test:*",
-    "tdd": "npm run test:karma -- --no-single-run",
-    "cover": "cat ./coverage/lcov/lcov.info | coveralls"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/mozilla/page-metadata-parser.git"
-  },
-  "keywords": [
-    "page",
-    "metadata",
-    "parser"
-  ],
```

Comment:
Most of this noise was just me running [**fixpack**](http://npm.im/fixpack) which likes to A-z things.